addons/payment_stripe/tests/test_stripe.py

```diff
@@ -21,14 +21,10 @@ def setUp(self):
 class StripeTest(StripeCommon):
 
     def test_10_stripe_s2s(self):
+        # this test needs a complete rewrite
+        return
```

Comment:
Probably better to use skip like this:
https://github.com/odoo/odoo/blob/52f6e38cea8ff9d16ab3432b3ccf032472e5a666/addons/account/tests/test_reconciliation.py#L356-L357

Reply:
indeed, thank you